src/components/common/names-capture.vue

```diff
@@ -539,7 +539,12 @@ export default class NamesCapture extends Mixins(CommonMixin) {
   }
 
   get entityTypeText (): string {
-    return (this.entity_type_cd === EntityType.CC) ? 'Community Contribution Company' : 'Cooperative'
+    if (this.entity_type_cd === EntityType.SO) {
+      return 'Society'
+    } else if (this.entity_type_cd === EntityType.CC) {
+      return 'Community Contribution Company'
+    }
+    return 'Cooperative'
```

Comment:
Instead of assuming the last possibility is coop, how about checking "if coop" for that case and adding a different fallback value?

Reply:
Maybe a fallback of "business" ?

Reply:
Modified the code. Please review

Reply:
Very nice! Thanks!